<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBug</class>
 <widget class="QMainWindow" name="AddBug">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>872</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AddBug</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#QLabel{

font-size: 15px;
font-family: Arial;
}

#QPushButton{
	font-size: 12px;
	min-width:50px;
	max-width:50px;
	max-height:13px;
	max-height:13px;
	padding:3px;
	
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="2" alignment="Qt::AlignLeft">
     <widget class="QPushButton" name="addBug_SubmitButton">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">gridline-color: rgb(58, 139, 250);

color:white;
</string>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>880</width>
        <height>330</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Add New Bug</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="addBug_TitleLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="addBug_titleLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="AddBug_ApplicationLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Application</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="addBug_DescriptionLabel">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Description</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2" rowspan="2">
        <widget class="QTextEdit" name="addBug_DescriptionTextEdit">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>180</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>180</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="addBug_ApplicationQCombo">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>12</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QGroupBox" name="addBug_ExtraOptionsFrame">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="addBug_SeverityCombo_2">
         <item>
          <property name="text">
           <string>Minor</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Major</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Neutral</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="addBug_FileUploadLabel">
         <property name="text">
          <string>Upload A file</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="addBug_PriorityCombo_2">
         <item>
          <property name="text">
           <string>Low</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Medium</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>High</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="addBug_VersionLabel">
         <property name="text">
          <string>Version</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="addBug_versionCombo"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="addBug_ComponentLabel">
         <property name="text">
          <string>Component</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Severity</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="addBug_PlatformLabel">
         <property name="text">
          <string>Platform</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="addBug_PlatformCombo"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Priority</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="addBug_WalkthroughLabel">
         <property name="text">
          <string>Walkthrough</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="3">
        <widget class="QComboBox" name="addBug_fileUploadCombo"/>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="addBug_componentsCombo"/>
       </item>
       <item row="1" column="3" rowspan="5">
        <widget class="QTextEdit" name="addBug_WalkthroughTextEdti"/>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="addBug_uploadFileButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
gridline-color: rgb(53, 135, 250);
color:white;</string>
         </property>
         <property name="text">
          <string>Upload File</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="addBug_RemoveFileButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Remove File</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>addBug_WalkthroughTextEdti</zorder>
      <zorder>addBug_WalkthroughLabel</zorder>
      <zorder>addBug_uploadFileButton</zorder>
      <zorder>addBug_RemoveFileButton</zorder>
      <zorder>addBug_fileUploadCombo</zorder>
      <zorder>addBug_FileUploadLabel</zorder>
      <zorder>addBug_PriorityCombo_2</zorder>
      <zorder>addBug_ComponentLabel</zorder>
      <zorder>addBug_PlatformCombo</zorder>
      <zorder>label</zorder>
      <zorder>addBug_PlatformLabel</zorder>
      <zorder>addBug_SeverityCombo_2</zorder>
      <zorder>addBug_VersionLabel</zorder>
      <zorder>label_2</zorder>
      <zorder>addBug_versionCombo</zorder>
      <zorder>addBug_componentsCombo</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox</zorder>
      <zorder>groupBox</zorder>
     </widget>
    </item>
    <item row="5" column="1" alignment="Qt::AlignRight">
     <widget class="QPushButton" name="addBug_cancelButton">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="addBug_OptionsButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Add Optional Information</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>872</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
